app/models/item.rb

```diff
@@ -0,0 +1,19 @@
+class Item < ApplicationRecord
+  belongs_to :user
+  has_one_attached :images
```

Comment:
```suggestion
  has_one_attached :image
```
railsの命名規則において、`has_one_attached`は1つのファイルを追加するために使用します。
そのため、imageは単数系にしましょう。

Reply:
修正しました